package.json

```diff
@@ -22,28 +22,26 @@
         "typescript": "4.5.5"
     },
     "devDependencies": {
-        "@babel/core": "7.17.5",
-        "@babel/plugin-proposal-nullish-coalescing-operator": "7.16.7",
-        "@babel/plugin-proposal-optional-chaining": "7.16.7",
-        "@babel/preset-env": "7.16.11",
-        "@babel/preset-typescript": "7.16.7",
+        "@swc/core": "^1.2.143",
```

Comment:
I'm also seeking to convert Docusaurus to ESM (https://github.com/facebook/docusaurus/issues/6520) and I've just been figuring out how to make Jest happy! Is this Babel -> SWC necessary, or is it just a quality-of-life improvement?

Reply:
Just quality of life, yeah. I'll probably split these changes out to make the PR and blog post more clear.

Reply:
https://github.com/typescript-eslint/tslint-to-eslint-config/issues/1366